compliance/github/repository_branch_protection/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v3.0
```

Comment:
Please revert the version to v2.3, 3.0 is not a valid version.

Reply:
Also, the version would need to be updated within the `info` statement in the Policy Template itself.

Reply:
I fixed the versions.